### DISCUSSION TITLE
isRefetching state

### DISCUSSION THREAD

**Discussion by BitPhinix** [urql-graphql/urql#3305] [Q&A]
Is there any way to access an isRefetching state from the useQuery hook to implement pull-to-refresh?

**Comment by kitten** [comment]
Yes, you're still looking for `fetching`: https://formidable.com/open-source/urql/docs/basics/react-preact/#reexecuting-queries

If you only want to look at cases where you have data and are refetching do `result.data && result.fetching` or even `result.fetching && result.operation` even
